### ISSUE TITLE
mini vs full: There is no page indicating differences

### ISSUE THREAD

**Issue by Albirew** [cmderdev/cmder#1960]
### Purpose of the issue
- [ ] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [x] Question

### Description of the issue 
There is no page on website, wiki or readme clearly indicating differences between Mini and Full version.
readme erratically indicates that mini doesn't work out-of-the-box with pre-configured bash and that vendored git libraries are not included but that's all. So where do these additionnal 99.5Mb comes from?

### tl;dr
Documentation should include a proper table or list with what's in Full edition that is not in Mini.

**Comment by Stanzilla** [comment]
> Oooh yes! If you decide to use the slightly bigger git-for-windows version, you will have all Unix commands ready in PATH so that you can git init or cat instantly on every machine.

**Comment by Albirew** [comment]
OK, my bad, I did missed that part.

**Comment by Vichoko** [comment]
Yeah, i really don't know which one i should pick and can't find any information to take an informed decision.

**Comment by DRSDavidSoft** [comment]
More information explained over at #2715.
